smoke-sketches can randomly fail with parallel make

**Describe the bug**
On macOS at least, running `make -j4 smoke-sketches` randomly fails due to `install -d` trying to create a directory that already exists.

**To Reproduce**
- Remove the `$(OUTPUT_PATH)` directory (under `$TMPDIR`).
- `make -j4 smoke-sketches`

Example failure output:

```
install: install: mkdir /var/folders/d5/nzysv6bx0fz3lt46bhlv_bhw0000gn/T//kaleidoscope-tlyu/output: File exists
install: mkdir /var/folders/d5/nzysv6bx0fz3lt46bhlv_bhw0000gn/T//kaleidoscope-tlyu/output: File exists
mkdir /var/folders/d5/nzysv6bx0fz3lt46bhlv_bhw0000gn/T//kaleidoscope-tlyu/output: File exists
make[1]: *** [compile] Error 71
make[1]: *** [compile] Error 71
make[1]: *** [compile] Error 71
make: *** [examples/Devices/Keyboardio/Atreus] Error 2
make: *** Waiting for unfinished jobs....
make: *** [examples/Devices/EZ/ErgoDox] Error 2
make: *** [examples/Devices/KBDFans/KBD4x] Error 2
```

**Expected behavior**
The `make smoke-sketches` command should run to completion without errors, even when run in parallel using `make -j`.

**Environment (please complete the following information):**
 - OS: macOS
 - Version 10.15.7 (19H2026)
 - Device N/A

**Additional context**
It seems that `install -d` will normally succeed if the directory already exists. However, it's possible that a race condition occurs between when it checks for existence and when it attempts to create the directory.

--- a/etc/makefiles/sketch.mk
+++ b/etc/makefiles/sketch.mk
@@ -164,7 +164,7 @@ endif
 compile: kaleidoscope-hardware-configured
 
 
-	$(QUIET) install -d "${OUTPUT_PATH}"
+	-$(QUIET) install -d "${OUTPUT_PATH}"
 	$(QUIET) $(ARDUINO_CLI) compile --fqbn "${FQBN}" ${ARDUINO_VERBOSE} ${ccache_wrapper_property} ${local_cflags_property} \
 	  ${_arduino_local_libraries_prop} ${_ARDUINO_CLI_COMPILE_CUSTOM_FLAGS} \
 	  --library "${KALEIDOSCOPE_DIR}" \

